packages/@aws-cdk/aws-iam/lib/group.ts

```diff
@@ -97,12 +97,13 @@ export class Group extends Construct implements IPrincipal {
   /**
    * Adds an IAM statement to the default policy.
    */
-  public addToPolicy(statement: PolicyStatement) {
+  public addToPolicy(statement: PolicyStatement): boolean {
```

Comment:
Do we want to add an awslint rule to enforce the return type?

Reply:
Seems #weird that this returns a `boolean`, too. Based on signature, I would expect it to return one of:
1. `this`
2. `statement`
3. some `Policy[Document]`.